controller/volume_controller.go

```diff
@@ -432,7 +428,7 @@ func (vc *VolumeController) EvictReplicas(v *longhorn.Volume,
 	for _, replica := range rs {
 		if replica.Status.EvictionRequested == true &&
 			healthyCount == v.Spec.NumberOfReplicas {
-			if err = vc.replenishReplicas(v, e, rs); err != nil {
+			if err = vc.replenishReplicas(v, e, rs, ""); err != nil {
```

Comment:
Why we're using `err = ` instead of `err :=` here? And there are similar cases below.

Reply:
Thank you! Fixed